inputfiles/addedTypes.jsonc

```diff
@@ -679,6 +683,14 @@
                             "overrideType": "null"
                         }
                     }
+                },
+                "events": {
+                    "event": [
+                        {
+                            "name": "DOMContentLoaded",
```

Comment:
Could you move this to `WindowEventHandlers`, thanks!

Reply:
Moving the event from `Document` to `WindowEventHandlers` will remove it from document and add it to `<body>`, `<frameset>`, and `<svg>`, which I believe is incorrect.

The `DOMContentLoaded` event target is always the document object, so the only objects that will ever receive it are document and window.

Reply:
Oh, I thought this was a Window one.

>The DOMContentLoaded event target is always the document object, so the only objects that will ever receive it are document and window.

Makes sense.

But then I wonder why there are `Event` and `Events` redundancy now...

Reply:
From what I can tell reading the build code:

1. The `Event` interface is now getting included in the `webidl` variable because it's being referenced.
2. It then gets included in the `distinctETypeList` variable because its name ends with the string "Event".

I don't know if this is expected behaviour or a bug, but the resulting definitions aren't exactly _wrong_, as running `document.createEvent( "Event" )` and `document.createEvent( "Events" )` in a browser works fine.

Reply:
It's not wrong, right, but I don't see anything that includes "Events" in the code nor in the webref dependency 🤔

I'd like to investigate before merging this...

Reply:
The plural is being generated by the function `emitCreateEventOverloads`, which contains a list of events that have plurals.

Reply:
Ah right. That function generates `createEvent` signature based on `type` and there currently is no `"type": "Event"`, so the addition here.